wpiunits/src/test/java/edu/wpi/first/units/MeasureTest.java

```diff
@@ -343,5 +343,17 @@ void testIsNear() {
 
     assertTrue(Units.Feet.zero().isNear(Units.Millimeters.zero(), 0.001));
     assertFalse(Units.Feet.of(2).isNear(Units.Millimeters.of(0), 0.001));
+
+    var measureG = Units.Meters.of(1);
+    var measureH = Units.Meters.of(1.2);
+
+    assertTrue(measureG.isNear(measureH, Units.Millimeters.of(300)));
+    assertFalse(measureG.isNear(measureH, Units.Centimeters.of(10)));
+
+    measureG = measureG.negate();
+    measureH = measureH.negate();
+
+    assertTrue(measureG.isNear(measureH, Units.Millimeters.of(300)));
```

Comment:
This should be a separate test, since it's checking a different function. Also, please rename the variables to be more descriptive

Reply:
7bced716672b17d4fe50e017ab7271d200a40d47
d5dee3ed88137bf58320c5f59af204b6cf92a497
5665845df1aa19ba48b808102235fe5a4b47c23b
Is this good?